src/v/cluster/members_backend.cc

```diff
@@ -263,7 +266,11 @@ ss::future<> members_backend::calculate_reallocations_after_recommissioned(
 
 ss::future<std::error_code> members_backend::reconcile() {
     // if nothing to do, wait
-    co_await _new_updates.wait([this] { return !_updates.empty(); });
+    try {
+        co_await _new_updates.wait([this] { return !_updates.empty(); });
+    } catch (const ss::broken_condition_variable& e) {
```

Comment:
nit: it will probably make sense to use `ssx::is_shutdown_exception` in combination with `seastar::coroutine::as_future`

Reply:
doesn't make sense given that _new_udpates is a cvar